lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -91,7 +91,7 @@
     /**
      * The hydration mode.
      *
-     * @var int
+     * @var int|string
```

Comment:
@Majkl578 WDYT about proposing hydration modes as enumerable types here as well? (Not related with this patch)